src/index.js

```diff
@@ -306,6 +313,14 @@ class InputElement extends React.Component {
     return this.focused;
   }
 
+  isCursorPastLastChar = () => {
+    const filledLength = getFilledLength(this.maskOptions, this.value);
+    const maxPosition = getRightEditablePosition(this.maskOptions, filledLength);
+    const currentPos = this.getCursorPosition();
```

Comment:
For consistency it's better to name variables either `maxPosition` and `currentPosition` or `maxPos` and `currentPos`.